Specs/Request/Request.JSONP.js

```diff
@@ -52,4 +57,29 @@ describe('Request.JSONP', function(){
 
 	});
 
+	it('should clear the request callback map', function(){
```

Comment:
why don't we use `function(done){ ... done() ... }` for async but we have `waitsFor(<ambiguous number>, fn);`